src/logging.ts

```diff
@@ -0,0 +1,11 @@
+import winston from 'winston';
+import dotenv from 'dotenv';
+
+dotenv.config();
```

Comment:
dotenv.config should be called only once - the idea is that we have a config in config.ts and call it there. 
from there on out, nothing else will directly access process env. they will instead access the config

Reply:
Shoot I used dotenv.config() outside of config.js in a couple more places in the codebase. I'll remove them in this PR then?

Reply:
it's ok - do it after postgres merges in